CHANGELOG.md

```diff
@@ -240,6 +240,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed `AttributeError for `require_backward_grad_sync` when running manual optimization with sharded plugin ([#6915](https://github.com/PyTorchLightning/pytorch-lightning/pull/6915))
 
 
+- Fixed a bug where `LightningModule.training_epoch_end` was called after the `on_train_end_epoch` hook ([#6969](https://github.com/PyTorchLightning/pytorch-lightning/pull/6969))
```

Comment:
can we write these changes more densely, this is quite long and someone needs detail he can open the particular PR, see the chlog entry as kind of title/subtitle

Reply:
I personally think it's nicer to mention what was exactly fixed than just saying "Fix inconsistent outputs in on_*_end and *_end"